include/hse/flags.h

```diff
@@ -23,6 +23,7 @@ extern "C" {
 
 /* hse_kvs_cursor_create() flags */
 #define HSE_CURSOR_CREATE_REV (1u << 0)
+#define HSE_CURSOR_CREATE_SURVIVE_TXN (1u << 1)
```

Comment:
You are missing documentation in the comment for the `hse_kvs_cursor_create()` function. Are we lining up the definitions? Personally don't care, but might want to see what the put flags are doing

Reply:
Name seems aight. Don't forget your hse-python and hse-java PRs!